src/app/model/schema.ts

```diff
@@ -44,13 +44,16 @@ export class UserInterface {
   FIXME: need to ensure this is synced from firebase.auth ph number
   FIXME: where do we assert phone number formatting?
   FIXME: currently, always null */
-  phone?: number;
+  phone!: string;
```

Comment:
Have we checked if this type change affects any other components consuming this information?